hackclub.com.yaml

```
@@ -907,8 +907,8 @@ conduct-test:
     value: v=spf1 include:amazonses.com ~all
 constellation:
   - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+    type: A
+    value: 64.23.238.153.
```

Remove the dot at the end of the ip addr
```suggestion
    value: 64.23.238.153
```

---

hello fixed